recipes-images/packagegroups/packagegroup-provision-tpm.bb

```diff
@@ -0,0 +1,10 @@
+DESCRIPTION = "Packagegroup for PKCS11 with TPM2."
+LICENSE = "MIT"
+
+inherit packagegroup
+
+SUMMARY_packagegroup-pkcs11-tpm2 = "PKCS11 with TPM 2.0 support"
```

Comment:
This seems a little weird. Why is the summary of a different packagegroup changed here?

Reply:
this was not the intention. is changed